e2e/ibazel.go

```diff
@@ -43,16 +43,16 @@ func (i *IBazelTester) Build(target string) {
 	i.build(target, []string{})
 }
 
-func (i *IBazelTester) Run(target string) {
-	i.run(target, []string{})
+func (i *IBazelTester) Run(target string, bazelArgs ...string) {
```

Comment:
I'm terribly sorry, but could you also flip the argument ordering here. It should always be `bazelArgs, target, [targetArgs]`

Reply:
No problem. However then I see possible inconsistency with `.run()` that goes like `(target, bazelArgs, ibazelArgs)`

What pattern am I missing? :)

Also `.Run()` has it (at least for now) as _optional_.

Reply:
DONE. ✔️ PTAL.

Reply:
You're 100% right that [`.run`](https://github.com/bazelbuild/bazel-watcher/blob/9a15dd18d8312b3b2770afd27564009296bf25fb/e2e/ibazel.go#L160) should be flipped around. Maybe it should even be a dictionary that maps from flag to value, but the reason to be persnickety about this is because it is part of the public interface. `.run` is internal to this function and I can fix it later without breaking anyone.

Thanks again for fixing this